Mage.Sets/src/mage/cards/c/CaseOfTheStashedSkeleton.java

```diff
@@ -0,0 +1,149 @@
+
+/**
+ *
+ * @author DominionSpy
+ */
+public final class CaseOfTheStashedSkeleton extends CardImpl {
+
+    static final FilterPermanent filter = new FilterControlledCreaturePermanent(SubType.SKELETON, "You control no suspected Skeletons");
+
+    static {
+        filter.add(SuspectedPredicate.instance);
+    }
+
+    public CaseOfTheStashedSkeleton(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{1}{B}");
+
+        this.subtype.add(SubType.CASE);
+
+        // When this Case enters the battlefield, create a 2/1 black Skeleton creature token and suspect it.
+        Ability initialAbility = new EntersBattlefieldTriggeredAbility(
+                new CaseOfTheStashedSkeletonEffect());
+        // To solve -- You control no suspected Skeletons.
+        Condition toSolveCondition = new PermanentsOnTheBattlefieldCondition(
+                filter, ComparisonType.EQUAL_TO, 0, true);
+        // Solved -- {1}{B}, Sacrifice this Case: Search your library for a card, put it into your hand, then shuffle. Activate only as a sorcery.
+        Ability solvedAbility = new ConditionalActivatedAbility(
+                new SearchLibraryPutInHandEffect(new TargetCardInLibrary(), false)
+                        .setText("Search your library for a card, put it into your hand, then shuffle. Activate only as a sorcery."),
+                new ManaCostsImpl<>("{1}{B}"),
+                SolvedSourceCondition.SOLVED)
+                .setTiming(TimingRule.SORCERY);
+        solvedAbility.addCost(new SacrificeSourceCost().setText("sacrifice this Case"));
+
+        this.addAbility(new CaseAbility(initialAbility, toSolveCondition, solvedAbility)
+                .addHint(new CaseOfTheStashedSkeletonHint(toSolveCondition)));
+    }
+
+    private CaseOfTheStashedSkeleton(final CaseOfTheStashedSkeleton card) {
+        super(card);
+    }
+
+    @Override
+    public CaseOfTheStashedSkeleton copy() {
+        return new CaseOfTheStashedSkeleton(this);
+    }
+}
+
+class CaseOfTheStashedSkeletonEffect extends CreateTokenEffect {
+
+    CaseOfTheStashedSkeletonEffect() {
+        super(new CaseOfTheStashedSkeletonToken());
+        staticText = "create a 2/1 black Skeleton creature token and suspect it";
+    }
+
+    private CaseOfTheStashedSkeletonEffect(final CaseOfTheStashedSkeletonEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public CaseOfTheStashedSkeletonEffect copy() {
+        return new CaseOfTheStashedSkeletonEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        super.apply(game, source);
+        Permanent token = game.getPermanent(this.getLastAddedTokenIds().stream().findFirst().orElse(null));
+        if (token != null) {
+            token.setSuspected(true, game, source);
+            return true;
+        }
+        return false;
+    }
+}
+
+class CaseOfTheStashedSkeletonToken extends TokenImpl {
```

Comment:
each token needs to be its own class in `mage.game.permanent.token`

Reply:
Do you know of any other instances of this token in MKM? If not, could I create a new token through `CreatureToken`?

Reply:
Nope. Token class must be used for public tokens only (if it has token image).

CreatureToken for dynamic tokens.